src/SarifWeb/Views/ValidationUi/Index.cshtml

```diff
@@ -343,8 +358,24 @@
                             $ruleDescription.html(noErrorsText);
                         }
                     }
-                    resolve();
-                });
+                }
+
+                setToolbarEnabled(true);
+            }
+
+            function isGitHubDspRule(rule) {
+                // Rules in the range SARIF2017-SARIF2025 are used to validate SARIF files intended
+                // for upload to the GitHub Developer Security Portal (DSP).
+                let ruleNumber = rule.id.substring(5)
```

Comment:
Recommend `const` in these cases.

Reply:
Fixed here and one other occurrence. I think I got most of them.

---
In reply to: [467327257](https://github.com/microsoft/sarif-website/pull/115#discussion_r467327257) [](ancestors = 467327257)